### DISCUSSION TITLE
DataTable Edit Row

### DISCUSSION THREAD

**Discussion by Jean-Sty** [primefaces/community#1367] [PrimeVue] [answered]
Hi, Regarding the DataTable Edit Row component, how I could change the 'quantity' value of maybe 'code' for such data type below?
{
                    id: '1001',
                    code: 'nvklal433',
                    name: 'Black Watch',
                    description: 'Product Description',
                    image: 'black-watch.jpg',
                    price: 72,
                    category: 'Accessories',
                    quantity: 61,
                    inventoryStatus: 'INSTOCK',
                    rating: 4,
                    orders: [
                        {
                            id: '1001-0',
                            productCode: 'nvklal433',
                            date: '2020-05-14',
                            amount: 72,
                            quantity: 1,
                            customer: 'Maisha Jefferson',
                            status: 'DELIVERED'
                        },
                        {
                            id: '1001-1',
                            productCode: 'nvklal433',
                            date: '2020-02-28',
                            amount: 144,
                            quantity: 2,
                            customer: 'Octavia Murillo',
                            status: 'PENDING'
                        }
                    ]
                },
                {
                    id: '1002',
                    code: 'zz21cz3c1',
                    name: 'Blue Band',
                    description: 'Product Description',
                    image: 'blue-band.jpg',
                    price: 79,
                    category: 'Fitness',
                    quantity: 2,
                    inventoryStatus: 'LOWSTOCK',
                    rating: 3,
                    orders: [
                        {
                            id: '1002-0',
                            productCode: 'zz21cz3c1',
                            date: '2020-07-05',
                            amount: 79,
                            quantity: 1,
                            customer: 'Stacey Leja',
                            status: 'DELIVERED'
                        },
                        {
                            id: '1002-1',
                            productCode: 'zz21cz3c1',
                            date: '2020-02-06',
                            amount: 79,
                            quantity: 1,
                            customer: 'Ashley Wickens',
                            status: 'DELIVERED'
                        }
                    ]
                },
                {
                    id: '1003',
                    code: '244wgerg2',
                    name: 'Blue T-Shirt',
                    description: 'Product Description',
                    image: 'blue-t-shirt.jpg',
                    price: 29,
                    category: 'Clothing',
                    quantity: 25,
                    inventoryStatus: 'INSTOCK',
                    rating: 5,
                    orders: []
                },

**Comment by tugcekucukoglu** [comment, marked as answer]
There is an example for this: https://primevue.org/datatable/#row_edit
